### ISSUE TITLE
Disconnected from server

### ISSUE THREAD

**Issue by ikey-ru** [thomasnordquist/MQTT-Explorer#529]
Connected 1 time normally.
I closed and opened application. Not connect. Why?
<img width="1369" alt="Снимок экрана 2021-03-24 в 00 22 40" src="https://user-images.githubusercontent.com/47115676/112222596-cd47ee80-8c39-11eb-9e18-94a17dc62c7e.png">

**Comment by ikey-ru** [comment]
Version 0.4.0 crashes on startup. MacOs Big Sur

**Comment by ilioSS53** [comment]
hello found the same problem ?
No help yet .
It is a pitty.
looking for something else.

**Comment by ikey-ru** [comment]
> hello found the same problem ? No help yet . It is a pitty. looking for something else.

Yes, need disable anonymous connections and enable listener in config file mqtt server.

**Comment by brunomacatronica** [comment]
Someboby? Same issue...

**Comment by EnziinSystem** [comment]
Version 0.4.0-beta1

After I turn off the secured MQTT

```
allow_anonymous true
listener 1883
```

And restart

`sudo systemctl restart mosquitto`

It works and connected to MQTT Explorer

**Comment by ripvega** [comment]
> Version 0.4.0-beta1
> 
> After I turn off the secured MQTT
> 
> ```
> allow_anonymous true
> listener 1883
> ```
> 
> And restart
> 
> `sudo systemctl restart mosquitto`
> 
> It works and connected to MQTT Explorer

I'm getting this same issue since the lastest update of Home Assistant.

The Mosquitto HA docs now say:

Note: This add-on does not support anonymous logins; all connections must use a username/password to connect. allow_anonymous true nor any anonymous ACLs will not work with this add-on.

Is there another way around this?
